demo/src/main/res/values/strings.xml

```diff
@@ -163,6 +163,16 @@
     <string name="component_dialog_action_confirm">Accept</string>
     <string name="component_dialog_action_ok">OK</string>
 
+    <!-- Component FAB -->
+    <string name="component_floating_action_buttons">Buttons: FAB</string>
```

Comment:
FAB is located after the dialogs in the components list but according to the title, it should be right after the buttons.

Maybe we should order components by their title in `ComponentsScreen`.